collection-scripts/gather_network_logs_basics

```diff
@@ -84,7 +113,7 @@ function gather_ovn_kubernetes_data {
     echo "INFO: LEGACY MODE"
     gather_ovn_kubernetes_data_legacy_mode
   fi
-
+  gather_ovn_kubernetes_ipsec_data
```

Comment:
maybe call this only when ipsec is enabled on clusters? so check the API first to see if IPSEC is set?

Reply:
yes @tssurya, done. collected mg report with changes, can see ipsec log being collected.

```
$ ls -l network_logs/ipsec/
total 20
-rw-r--r--. 1 peri peri 1670 Oct 11 05:24 ovn-ipsec-host-2qx2t_ipsec.conf
drwxr-xr-x. 3 peri peri   60 Oct 11 05:26 ovn-ipsec-host-2qx2t_ipsec.d
-rw-r--r--. 1 peri peri 1670 Oct 11 05:24 ovn-ipsec-host-8mztz_ipsec.conf
drwxr-xr-x. 3 peri peri   60 Oct 11 05:26 ovn-ipsec-host-8mztz_ipsec.d
-rw-r--r--. 1 peri peri 1670 Oct 11 05:24 ovn-ipsec-host-lq6j7_ipsec.conf
drwxr-xr-x. 3 peri peri   60 Oct 11 05:26 ovn-ipsec-host-lq6j7_ipsec.d
-rw-r--r--. 1 peri peri 1670 Oct 11 05:24 ovn-ipsec-host-p9h9f_ipsec.conf
drwxr-xr-x. 3 peri peri   60 Oct 11 05:26 ovn-ipsec-host-p9h9f_ipsec.d
drwxr-xr-x. 2 peri peri  166 Oct 11 05:26 status
drwxr-xr-x. 2 peri peri  166 Oct 11 05:26 trafficstatus
drwxr-xr-x. 2 peri peri 4096 Oct 11 05:26 xfrm
$ ls -l network_logs/ipsec/status/
total 1876
-rw-r--r--. 1 peri peri 529526 Oct 11 05:24 ovn-ipsec-host-2qx2t.log
-rw-r--r--. 1 peri peri      0 Oct 11 05:24 ovn-ipsec-host-46zh7.log
-rw-r--r--. 1 peri peri 457694 Oct 11 05:24 ovn-ipsec-host-8mztz.log
-rw-r--r--. 1 peri peri 460203 Oct 11 05:24 ovn-ipsec-host-lq6j7.log
-rw-r--r--. 1 peri peri 466707 Oct 11 05:24 ovn-ipsec-host-p9h9f.log
$ ls -l network_logs/ipsec/trafficstatus/
total 644
-rw-r--r--. 1 peri peri 179560 Oct 11 05:24 ovn-ipsec-host-2qx2t.log
-rw-r--r--. 1 peri peri      0 Oct 11 05:24 ovn-ipsec-host-46zh7.log
-rw-r--r--. 1 peri peri 158536 Oct 11 05:24 ovn-ipsec-host-8mztz.log
-rw-r--r--. 1 peri peri 156801 Oct 11 05:24 ovn-ipsec-host-lq6j7.log
-rw-r--r--. 1 peri peri 158297 Oct 11 05:24 ovn-ipsec-host-p9h9f.log
$ ls -l network_logs/ipsec/xfrm/
total 3796
-rw-r--r--. 1 peri peri   4698 Oct 11 05:24 ovn-ipsec-host-2qx2t_policy.log
-rw-r--r--. 1 peri peri 972455 Oct 11 05:24 ovn-ipsec-host-2qx2t_state.log
-rw-r--r--. 1 peri peri      0 Oct 11 05:24 ovn-ipsec-host-46zh7_policy.log
-rw-r--r--. 1 peri peri      0 Oct 11 05:24 ovn-ipsec-host-46zh7_state.log
-rw-r--r--. 1 peri peri   4698 Oct 11 05:24 ovn-ipsec-host-8mztz_policy.log
-rw-r--r--. 1 peri peri 962893 Oct 11 05:24 ovn-ipsec-host-8mztz_state.log
-rw-r--r--. 1 peri peri   4698 Oct 11 05:24 ovn-ipsec-host-lq6j7_policy.log
-rw-r--r--. 1 peri peri 949430 Oct 11 05:24 ovn-ipsec-host-lq6j7_state.log
-rw-r--r--. 1 peri peri   4698 Oct 11 05:24 ovn-ipsec-host-p9h9f_policy.log
-rw-r--r--. 1 peri peri 961124 Oct 11 05:24 ovn-ipsec-host-p9h9f_state.log
$ ls -l network_logs/ipsec/ovn-ipsec-host-2qx2t_ipsec.d
total 8
-rw-r--r--. 1 peri peri  276 Oct 11 05:24 cno.conf
-rw-r--r--. 1 peri peri 2858 Oct 11 05:24 openshift.conf
drwxr-xr-x. 2 peri peri  120 Oct 11 05:26 policies
```
the logs from node which has `ovn-ipsec-host-46zh7 `pod not having any data because ipsec is broken there due to a change in crypto policy setting, so that can be safely ignored.